node/npm/glide/index.ts

```diff
@@ -17,7 +18,9 @@ function loadNativeBinding() {
                     nativeBinding = require("@scope/glide-for-redis-linux-x64");
                     break;
                 case "arm64":
-                    nativeBinding = require("@scope/glide-for-redis-linux-arm64");
+                    nativeBinding = existsSync("./node_modules/@scope/glide-for-redis-linux-musl-arm64")
```

Comment:
Can you explain why using `existsSync`?

Reply:
Pretty sure there's nothing obvious in `process` that distinguishes between linux and linux-musl

Reply:
When this code will be called, it will be running on the target host that tries to install glide. If the host is running on musl, only the glide-for-redis-linux-musl-X dependency will be installed on the target host, and therefore he's using the existsSync check to find out which OS are we running on. but, as I said above - I think this isn't the right way to do it

Reply:
oh, sorry, both distributions will be installed. @ofirsnb Based on [os](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#os):
> The host operating system is determined by process.platform

If process.platform is 'linux' both for musl and not-musl platforms, it means that we install both musl and the glibc linux dependencies on every linux host. That means that with your current implementation (checking if the glide-for-redis-linux-musl-X dependency exists), we'll always require the musl dependency, also on glibc hosts.

Reply:
You've also changed in the node-create-package file that ${node_os} will be override with 'linux_musl', however  npm only accepts the `process.platform` output, which should be `linux`.

Reply:
You're correct @barshaul ,   missed the `os`.
This method should be changed.
I suggest using this:
https://github.com/ZingerLittleBee/system-status/blob/main/index.js#L10
as it's also the exact way `napi` uses to determine `musl`.